test/client/submitSignedTransaction.ts

```diff
@@ -0,0 +1,54 @@
+import { assert } from 'chai'
+
+import { Transaction } from 'xrpl-local/models/transactions'
+
+import rippled from '../fixtures/rippled'
+import { setupClient, teardownClient } from '../setupClient'
+
+describe('client.submitSignedTransaction', function () {
```

Comment:
this should use the `addressTests` setup so it checks our X-Address infrastructure as well